### PR TITLE
Fix missing Rust dependency in the dockerfile

### PR DESCRIPTION
**Pull request by jeremi**
## Description

Rust is missing from the build, Adding it.

## Type of change

- [ ] New features (non-breaking change).
- [X] Bug fix (non-breaking change).
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected).

## Checklist

- [X] I have performed a self-review of my code.
- [ ] I have added thorough tests if it is a core feature.
- [ ] There is a reference to the original bug report and related work.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] The feature is well documented.

### CODE REVIEW

**Comment by ducminhle** [comment]
Are you using macbook arm chipset, @jeremi?

**Comment by jeremi** [comment]
Yes, I do.

There is one issue that I got, which is Graphrag is not working to index, but the regular RAG is working. But I think it is unrelated to this.

File: Dockerfile [hunk 1]
```diff
@@ -21,6 +21,10 @@ RUN apt-get update -qqy && \
     && apt-get autoremove \
     && rm -rf /var/lib/apt/lists/*
 
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```
Comment (ducminhle) [inline Dockerfile]: I have tested; it failed. Please add `curl` at line 14, @jeremi.
And you should update commit messages and PR title following [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

Comment (niko-nnkn) [reply Dockerfile]: @jeremi , @ducminhle, can we directly install the `cargo` by using the apt command.
```
apt-get install -y cargo
```

**Comment by jeremi** [comment]
ok, feel free to edit this PR or redo it.

**Comment by cin-niko** [comment]
Sorry @jeremi, when rebasing, I accidentally pushed the wrong `main` branch to the remote branch. This led to GitHub automatically closing the PR (because there were no changes) 🙇‍♂
Please update and reopen PR or continue here https://github.com/Cinnamon/kotaemon/pull/226

**Pull request closed without merging**